tests/testthat/test-install.R

```diff
@@ -26,8 +26,9 @@ test_that("safe_build_package fails appropriately with pkgbuild", {
   on.exit(unlink(out, recursive = TRUE))
   opts <- c("--no-resave-data", "--no-manual", "--no-build-vignettes")
   expect_error(
-    safe_build_package(test_path("invalidpkg"), build_opts = opts , out, quiet = TRUE, use_pkgbuild = TRUE),
-    "System command error"
+    safe_build_package(test_path("invalidpkg"), build_opts = opts, build_manual = FALSE, build_vignettes = FALSE, out, quiet = TRUE, use_pkgbuild = TRUE),
+    "System command error",
+    class = "system_command_status_error"
```

Comment:
This just silences a warning from the newer testthat, as this is a classed error message.